src/index.html

```diff
@@ -19,8 +19,8 @@ <h1 id="leaderBoard">Leaderboard</h1>
         </div>
         <div id="rightDiv">
             <h2 id="scoreHeader">Add your score</h2>
-            <input type="text" id="nameInput" placeholder="Your name">
-            <input type="text" id = "scoreInput" placeholder="Your score">
+            <input type="text" class="input" id="nameInput" placeholder="Your name">
+            <input type="text" class="input" id = "scoreInput" placeholder="Your score">
```

Comment:
[ Optional Change ]

1.  It would be nice if you could change the type of the score input field to a number instead of text so that it takes only number type values from the user. 👍🏽 

<img width="490" alt="Screenshot 2022-12-24 at 2 24 53 PM" src="https://user-images.githubusercontent.com/50941074/209440132-20d0beaf-1f7e-4642-8704-e366503a907a.png">